docs/user/data_collection.rst

```diff
@@ -0,0 +1,89 @@
+Data Collection
+===============
+This document describes the types of data that Shield collects.
+
+Uptake
+------
+Shield monitors the execution of :ref:`recipes <recipes>` and reports to
+`Firefox Telemetry`_ the amount of successful and failed runs. This data is
+reported as a `main-ping histogram`_ under the
+``UPTAKE_REMOTE_CONTENT_RESULT_1`` key. The type and format of this histogram
```

Comment:
Nit: I was confused here for a second because I was looking for namespacing of shield here and didn't find any. It's documented further down, but maybe just mentioning there's further key-ing is helpful.